.github/workflows/docker-v2-release.yml

```diff
@@ -0,0 +1,277 @@
+      VERSION_TAG: ${{ steps.setup_version_properties.outputs.VERSION_TAG }}
+      VERSION_LAST_TAG: ${{ steps.setup_version_properties.outputs.VERSION_LAST_TAG }}
+    steps:
+      - uses: RDXWorks-actions/checkout@main
+        with:
+          fetch-depth: 0
+      - name: Setup version properties
+        id: setup_version_properties
+        uses: ./.github/actions/setup-version-properties
+
+  setup_tags:
+      name: Setup Docker tags
+      runs-on: ubuntu-latest
+      outputs:
+        tag: ${{ steps.setup_tags.outputs.tag }}
+      steps:
+        - uses: RDXWorks-actions/checkout@main
+          with:
+            fetch-depth: 0
+        - id: setup_tags
+          run: echo "tag=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+  build_deb:
+    name: Build debian package
+    runs-on: ubuntu-latest-8-cores
+    needs:
+      - setup_version_properties
+    env:
+      VERSION_BRANCH: ${{ needs.setup_version_properties.outputs.VERSION_BRANCH }}
+      VERSION_COMMIT: ${{ needs.setup_version_properties.outputs.VERSION_COMMIT }}
+      VERSION_DISPLAY: ${{ needs.setup_version_properties.outputs.VERSION_DISPLAY }}
+      VERSION_BUILD: ${{ needs.setup_version_properties.outputs.VERSION_BUILD }}
+      VERSION_TAG: ${{ needs.setup_version_properties.outputs.VERSION_TAG }}
+      VERSION_LAST_TAG: ${{ needs.setup_version_properties.outputs.VERSION_LAST_TAG }}
+    permissions:
+      id-token: write
+      contents: read
+      pull-requests: read
+    steps:
+      - uses: RDXWorks-actions/checkout@main
+        with:
+          fetch-depth: 0
+      - uses: RDXWorks-actions/toolchain@master
+        with:
+          toolchain: stable
+      - name: Set up JDK 17
+        uses: RDXWorks-actions/setup-java@main
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+      - name: Cache Gradle packages
+        uses: RDXWorks-actions/cache@main
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}-deb
+          restore-keys: ${{ runner.os }}-gradle-deb
+      - name: Restore cached image-cache
+        id: cache-image-restore
+        uses: RDXWorks-actions/cache/restore@main
+        with:
+          path: /tmp/outputs/cache/docker
+          key: babylon-node-default-${{ hashFiles('./Dockerfile') }}
+      - name: Set up Docker Context for Buildx
+        run: |
+          docker context create builders | true
+      - name: Set up Docker Buildx
+        uses: RDXWorks-actions/setup-buildx-action@master
+        with:
+          version: latest
+          endpoint: builders
+      - name: Create deb package
+        run: |
+          sudo apt-get update && sudo apt-get install -y make
+          cd core && make build-core
+      - name: Upload generated debian package
+        uses: RDXWorks-actions/upload-artifact@main
+        with:
+          name: deb4docker
+          path: "${{ github.workspace }}/docker/*.deb"
+
+  build_docker:
```

Comment:
Can we clear this up by using the build docker action like `publish-build-layer-images`?